buildchain/buildchain/targets/image.py

```diff
@@ -69,3 +70,8 @@ def uncompressed_filename(self) -> Path:
     def tag(self) -> str:
         """Image tag."""
         return '{img.name}:{img.version}'.format(img=self)
+
+    def save_image(self, save_path: Path) -> Union[TaskError, bool]:
+        """Save a previously created and tagged image"""
+        saver = SaveCommand(tag=self.tag, save_path=save_path)
```

Comment:
`saver`? Why not `cmd` or `command` or since it's callable:

```Python
         do_save = SaveCommand(tag=self.tag, save_path=save_path) 
         return do_save()
```